src/cmds/core/other.py

```diff
@@ -18,12 +18,9 @@ def __init__(self, bot: Bot):
 
     @slash_command(guild_ids=settings.guild_ids, description="A simple reply stating hints are not allowed.")
     async def no_hints(
-        self, ctx: ApplicationContext,
-        channel: Option(GuildChannel, "The guild channel to send the message to.", required=False) = None
-    ) -> Message:
+        self, ctx: ApplicationContext
+    ):
```

Comment:
add the `Message` type hint here
```suggestion
    ) -> Message:
```